loopy/schedule/__init__.py

```diff
@@ -643,11 +643,23 @@ def get_insns_in_topologically_sorted_order(kernel):
         for dep in insn.depends_on:
             rev_dep_map[dep].add(insn.id)
 
+    insn_id_to_feature_id = {}
+    insn_features = {}
```

Comment:
Document the intention here. (I'm guessing: don't redundantly create these "feature" objects. But why? To avoid memory use?)